### PR TITLE
fix(DDI-1111): fix GitHub action to add VSCode html-data json to buil…

### PR DESCRIPTION
**Pull request by vanessatran-ddi**
### Description
Adding an extra step to copy html-data.json for VS Code into the build folder. It will fix the github action.

### CODE REVIEW

**Comment by chrisolsen** [comment]
I am not sure this will be needed once the issue with the web-component's package.json file is fixed.

File: .github/workflows/alpha-release-ci.yml [hunk 1]
```diff
@@ -41,6 +41,8 @@ jobs:
         run: |
           npx nx affected --target=build --base=${{ steps.last_successful_commit.outputs.commit_hash }} --prod --with-deps
           npx nx affected --target=post --base=${{ steps.last_successful_commit.outputs.commit_hash }}
+      - name: Update VsCode documentation
+        run: npm run postbuild:web-components
```
Comment (chrisolsen) [inline .github/workflows/alpha-release-ci.yml:45]: Can you rename the task in the package.json to something like `build:vscode-blah` just to make things more clear?

**Review by chrisolsen — DISMISSED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**